src/MeetingsSDKAdapter.js

```diff
@@ -344,19 +362,19 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
       });
 
       // Listen to attach mediaStream source objects to the existing meeting
-      const meetingWithReadyEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_READY).pipe(
+      const meetingWithMediaReadyEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_READY).pipe(
         filter((event) => MEDIA_EVENT_TYPES.includes(event.type)),
         map((event) => this.attachMedia(ID, event)),
         map(() => this.meetings[ID])
       );
 
-      // Listen to mute event to return the meeting object
-      const meetingWithLocalMuteEvents$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_MUTE).pipe(
+      // Listen to update event to return the meeting object
+      const meetingWithLocalUpdateEvents$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_UPDATE).pipe(
```

Comment:
I thought you said updating the meeting object wasn't necessary? Since the meeting object reference that the component will have would also get updated

Reply:
we still need to push the data so the UI knows there had been a change.

Reply:
will take care of this in upcoming PR